src/sandbox/components/screens/call-functions/components/available-functions.tsx

```diff
@@ -0,0 +1,275 @@
+// @ts-nocheck
+import React from 'react';
+import { Box, BoxProps, color, Flex, Grid, Stack, transition } from '@stacks/ui';
+import { useRecoilValue, useRecoilState, useSetRecoilState } from 'recoil';
+import { Input } from '@components/inputs';
+import { useFormik } from 'formik';
+import { Button } from '@components/button';
+import { border } from '@common/utils';
+import { Caption, Text } from '@components/typography';
+import { currentFunctionState, readOnlyState, readOnlyResponseState } from '@sandbox/store/sandbox';
+import { useApiServer } from '@common/hooks/use-api';
+import { Section } from '@components/section';
+import { Badge } from '@components/badge';
+import { IconButton } from '@components/icon-button';
+import { encodeClarityValue, getTypeString, tupleCV } from '@stacks/transactions';
+import { ArrowRightIcon } from '@components/icons/arrow-right';
+import ApiIcon from 'mdi-react/ApiIcon';
+import FunctionIcon from 'mdi-react/FunctionIcon';
+import { LoadingPanel } from '@components/loading-panel';
+
+import { useUser } from '@sandbox/hooks/use-user';
+import { parseReadOnlyResponse } from '@sandbox/common';
+import { CodeBlock } from '@components/code-block';
+import { handleContractCall } from '@sandbox/common/connect-functions';
+import { useContractInterface } from '@sandbox/components/screens/call-functions/components/use-contract-interface';
+
+const ArgLine = ({ name, type, handleChange, placeholder = name, ...rest }: any) => (
+  <Box width="100%" {...rest}>
+    <Input
+      width="100%"
+      type={getTypeString(type).includes('int') ? 'number' : 'text'}
```

Comment:
Honestly I've had such issues with number inputs for ints, number doesn't help much if you can enter valid floats